pennylane/devices/default_tensor.py

```diff
@@ -226,8 +226,8 @@ class DefaultTensor(Device):
             `quimb's tensor_contract documentation <https://quimb.readthedocs.io/en/latest/autoapi/quimb/tensor/tensor_core/index.html#quimb.tensor.tensor_core.tensor_contract>`_.
             Default is ``"auto-hq"``.
         local_simplify (str): The simplification sequence to apply to the tensor network for computing local expectation values.
-            For a complete list of available simplification options, see the
-            `quimb's full_simplify documentation <https://quimb.readthedocs.io/en/latest/autoapi/quimb/tensor/tensor_core/index.html#quimb.tensor.tensor_core.TensorNetwork.full_simplify>`_.
+            At present, this can only be used with the TN method. For a complete list of available simplification options,
```

Comment:
```suggestion
            Currently, only the ``"tn"`` method is supported. For a complete list of available simplification options,
```

Optional suggestion 😄 .

Reply:
Thanks for the suggestion! 

Here we are using MPS and TN to indicate the 'Matrix Product State' and 'Tensor Network' methods (these have been defined before) in the `Keyword arguments` doc section. 

``"tn"`` and ``"mps"`` are only used to indicate the possible values of the `method` positional argument. Therefore, I chose TN to be consistent with the rest of the documentation.

The entire statement is:

```
The simplification sequence to apply to the tensor network for computing local expectation values. 

At present, this can only be used with the TN method. 

For a complete list of available simplification options, see the `quimb's full_simplify documentation. Default is ``"ADCRS"``.
```

If I replaced the second statement with: 

```
Currently, only the ``"tn"`` method is supported. 
```

I think that the connection with `local_simplify` would be less evident. What do you think?

Reply:
Ah, okay this makes sense! Sorry I missed that 😄